From 52242298f1b5f4762ef183a285de903b2ec1cf25 Mon Sep 17 00:00:00 2001
From: Liu Ying <Ying.Liu@freescale.com>
Date: Thu, 23 Apr 2015 16:31:38 +0800
Subject: [PATCH] MLK-10723-3 mxc IPUv3: common: Remove system sleep power
 management hooks

The system sleep power management hooks ipu_suspend/ipu_resume are actually
doing nothing, so let's remove them.

Signed-off-by: Liu Ying <Ying.Liu@freescale.com>
---
 drivers/mxc/ipu3/ipu_common.c | 13 -------------
 1 file changed, 13 deletions(-)

diff --git a/drivers/mxc/ipu3/ipu_common.c b/drivers/mxc/ipu3/ipu_common.c
index 2fa15e6..1eaf9fe 100644
--- a/drivers/mxc/ipu3/ipu_common.c
+++ b/drivers/mxc/ipu3/ipu_common.c
@@ -3309,18 +3309,6 @@ int ipu_ch_param_get_axi_id(struct ipu_soc *ipu, ipu_channel_t channel, ipu_buff
 EXPORT_SYMBOL(ipu_ch_param_get_axi_id);
 
 #ifdef CONFIG_PM
-static int ipu_suspend(struct device *dev)
-{
-	dev_dbg(dev, "ipu suspend.\n");
-	return 0;
-}
-
-static int ipu_resume(struct device *dev)
-{
-	dev_dbg(dev, "ipu resume.\n");
-	return 0;
-}
-
 int ipu_runtime_suspend(struct device *dev)
 {
 	release_bus_freq(BUS_FREQ_HIGH);
@@ -3339,7 +3327,6 @@ int ipu_runtime_resume(struct device *dev)
 
 static const struct dev_pm_ops ipu_pm_ops = {
 	SET_RUNTIME_PM_OPS(ipu_runtime_suspend, ipu_runtime_resume, NULL)
-	SET_SYSTEM_SLEEP_PM_OPS(ipu_suspend, ipu_resume)
 };
 #endif
 
-- 
1.8.0

